bin/apisix

```diff
@@ -609,6 +632,15 @@ local function init()
         sys_conf["worker_processes"] = "auto"
     end
 
+    local dns_resolver = sys_conf["dns_resolver"]
```

Comment:
need to update doc of dns: https://github.com/apache/incubator-apisix/blob/master/conf/config.yaml#L83

Reply:
done